packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/minimumHosts/controller.js

```diff
@@ -0,0 +1,7 @@
+import { BACKUP_MINIMUM_HOST_COUNT } from '../backup.constants';
+
+export default class {
+  constructor() {
+    this.minimumHosts = BACKUP_MINIMUM_HOST_COUNT;
+  }
+}
```

Comment:
Suggestion: You can get rid of the controller file, by passing the constant in the routing, and pass it into component bindings.

Reply:
Good one @JeremyDec . Thanks. This is done :)